deployment/src/main/java/io/quarkus/jsch/deployment/JSchSessionBuildItem.java

```diff
@@ -0,0 +1,35 @@
+package io.quarkus.jsch.deployment;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+public final class JSchSessionBuildItem extends MultiBuildItem {
```

Comment:
Please add a Javadoc to this Build Item

Reply:
fixed in last update, you can check if it's OK for you